lib/form/html.js

```diff
@@ -137,6 +138,10 @@ const addRequired = (html, input, index) => {
       // add required to input
       const reName = new RegExp(`name=['"]\\s*${inp.name}\\s*['"]`);
       render = render.replace(reName, `name="${inp.name}" aria-required="true" required`);
+
+      // Add required text to label
+      const reLabel = new RegExp('</label>');
+      render = render.replace(reLabel, `<mark class="mark-${required}">required to ${required}</mark></label>`);
```

Comment:
@Snugug do you mean this should be:
`required--publish`
or
`mark-required--publish`
?